### PR TITLE
fix: Scrolling  enabled

### LINKED ISSUE #77: Scroll function is not available on screen rotation
**Describe the bug**
On screen rotation, the scroll option is not available on the home screen and the screen is just set to some limited options.

**To Reproduce**
On home screen switch to landscape mode.

**Screenshots**
ezgif com-video-to-gif(3)

### PR DESCRIPTION
**Pull request by ShivangiSingh17**
Fixed #77 
On switching to landscape mode, the scrolling works for the home screen.

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,149 +1,153 @@
...
-        <LinearLayout
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_gravity="center"
-            android:layout_marginLeft="@dimen/margin_5"
-            android:layout_marginRight="@dimen/margin_5">
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="fill_parent"
+    android:layout_height="fill_parent">
+    <layout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (Chromicle) [inline app/src/main/res/layout/activity_main.xml]: Delete additional namespaces please

**Comment by yashk2000** [comment]
@ShivangiSingh17 any updates?

**Comment by ShivangiSingh17** [comment]
Will update soon..

**Comment by yashk2000** [comment]
@ShivangiSingh17 updates?

**Pull request closed without merging**